x-pack/test_serverless/api_integration/test_suites/common/index_management/create_enrich_policies.ts

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
Do these API integration tests already exist on stateful? We should probably have similar coverage there as well.